src/plugins/Plugin.ts

```diff
@@ -13,6 +18,11 @@ export type PluginContext = {
     record: RecordEvent;
     recordPageView: RecordPageView;
     getSession: GetSession;
+    getCurrentUrl: GetCurrentUrl;
+    getCurrentPage: GetCurrentPage;
+    getRequestCache: GetRequestCache;
+    incrementFetch: IncrementFetch;
+    decrementFetch: DecrementFetch;
```

Comment:
issue: I think once we move `TrackerPlugin` to the core logic (i.e., in the same folder as `PageManager`), these can be removed.